package.json

```diff
@@ -12,8 +15,9 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.1.2",
```

Comment:
Are you using Redux in this PR? I haven't had a chance to look at all of the changes, so maybe you are. Otherwise, I would suggest we hold off on bringing in Redux unless we really find a use case for it. React has built-in state management tooling like React Contexts and useReducer hooks that we can stick to unless we find we need something that handles  more complexity.  Or if it's just a tool you want to practice using more, that's fine as well. Just make sure to discuss it with your teammates as well.